src/plugins/intel_gna/transformations/convert_unaligned_concat_to_cnn.cpp

```diff
@@ -0,0 +1,535 @@
+    if (unalignedOutputBadShapeNode != nullptr) {
+        unalignedOutputBadShapeNodeName = unalignedOutputBadShapeNode->get_type_name();
+        if (unalignedOutputBadShapeNode->get_output_size() == 1) {
+            const auto unalignedOutputBadShapeNodeInputs = unalignedOutputBadShapeNode->output(0).get_target_inputs();
+            unalignedOutputBadShapeNodeInputsNumber = unalignedOutputBadShapeNodeInputs.size();
+        }
+    }
+
+    auto reshape_pattern_initial = CreateConstantU64V(std::vector<size_t> {1, totalElems});
+    auto initialReshape = std::make_shared<ngraph::opset7::Reshape>(unalignedOutputBadShape, reshape_pattern_initial, false);
+
+    // Example of concat's output
+    // (The scale is not kept)
+    //
+    // Aligned address                                 putAt                         Aligned                   Aligned
+    // X               putAtOffsetFromAligned            |                               X                        X
+    // X-----------Previous concat component-------------|---delayedCopySourceOffset-----X---effeciveOutElements--------------|--tail-----|
+    // X                                                 |            Unaligned concat component ------- totalElems ---                   |
+    std::shared_ptr < ov::Node > convolutionReshaped;
+    const auto ramainingOut = totalElems > delayedCopySourceOffset ? totalElems - delayedCopySourceOffset : 0u;
+    size_t effeciveOutElements = 0;
+    if (maxCnnCopyIn >= totalFilter && ramainingOut > 0) {
+        auto maxOutElements = maxCnnCopyIn - totalFilter + numberOfFilters;
+
+        // CNN input and output can be shorten if Affine filter based tail copy is needed
+        // Those outputs can be computed by the Affine filter based tail copy
+        size_t notNeededCnnOutputs = 0;
+        if (ramainingOut != maxOutElements) {
+            notNeededCnnOutputs = DownAlignTo(maxOutElements % elementsAligned, commonDivisor);
+        }
+        effeciveOutElements = maxOutElements - notNeededCnnOutputs;
+        const auto effectiveInElements = maxCnnCopyIn - notNeededCnnOutputs;
+
+        auto ss_for_cnn_in = AppendStridedSliceOn2DNode(initialReshape, 0, effectiveInElements);
+
+        auto reshape_pattern = CreateConstantU64V(std::vector<size_t> {1, 1, 1, effectiveInElements});
+        auto reshapeCocnatInput = std::make_shared<ngraph::opset7::Reshape>(ss_for_cnn_in, reshape_pattern, false);
+
+        auto filterCnn1DNW = getDelayedCopyCnnFilter(4, totalFilter, delayedCopySourceOffset);
+
+        auto filters = std::make_shared< ngraph::opset7::Constant>(outputType, ngraph::Shape{ 4, 1, 1, totalFilter }, filterCnn1DNW);
+
+        auto convolution = std::make_shared<ngraph::opset7::Convolution>(reshapeCocnatInput, filters, ngraph::Strides{ 1 , 4 },
+            ngraph::CoordinateDiff{}, ngraph::CoordinateDiff{}, ngraph::Strides{});
+
+        convolution->get_rt_info()["GNA_UAC"] = rtValue(1);
```

Comment:
What this mark means for convolution?

Reply:
it is already removed and replaced with LayerInfo::isConvolutionFromUnalignedConcatFilter()